kite-morphlines/kite-morphlines-phonenumber/pom.xml

```diff
@@ -0,0 +1,69 @@
+<!--
+
+    Copyright (C) 2014 Stratio (http://stratio.com)
```

Comment:
Ok! I'm following the steps to sign the CLA. All take care of the rest of your comments ASAP.